### PR TITLE
update readme to work with web console

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@jwforres review please.

File: examples/sample-app/README.md [hunk 1]
```diff
@@ -153,7 +163,7 @@ In addition to creating resources, you can delete resources based on IDs. For ex
 
   - List the existing services:
 
-        $ openshift kubectl get services
+        $ openshift cli get services
```
Comment (jwforres) [inline examples/sample-app/README.md]: missing --namespace flag on the rest of these

File: examples/sample-app/README.md [hunk 2]
```diff
@@ -61,37 +61,44 @@ All commands assume the `openshift` binary is in your path (normally located und
...
     You should see:
 
         "docker-registry server (dev) (v0.9.0)"
 
 
-7. Fork the [ruby sample repository](https://github.com/openshift/ruby-hello-world)
+7. Create a new project in OpenShift
 
-8. *Optional:* Add the following webhook to your new github repository:
+        $ openshift cli create Project -f project.json
```
Comment (jwforres) [inline examples/sample-app/README.md]: project.json isn't checked in under sample-app so this would fail

**Comment by bparees** [comment]
@jwforres good catches, comments addressed.

**Comment by jwforres** [comment]
LGTM

**Comment by bparees** [comment]
[merge]

**Comment by bparees** [comment]
[merge]

**Pull request merged**